dns/rdata.py

```diff
@@ -551,6 +552,8 @@ def _as_ipv4_address(cls, value):
             return dns.ipv4.canonicalize(value)
         elif isinstance(value, bytes):
             return dns.ipv4.inet_ntoa(value)
+        elif isinstance(value, ipaddress.IPv4Address):
+            return str(value)
```

Comment:
For consistency, this should probably be changed in the same way as the IPv6 case, even though it's pretty unlikely to matter in any case.